fpr2par/views.py

```diff
@@ -1004,37 +1024,31 @@ def businessRules():
     response["businessRules"] = []
     for rule in rules:
         command = fpr_commands.query.get(rule.command)
-        format = fpr_format_versions.query.get(rule.format)
-        if format.pronom_id:
-            formatName = format.pronom_id
-            if format.pronom_id[:3] == "arc":
+        file_format = fpr_format_versions.query.get(rule.format)
+        if format_filter != [] and file_format.pronom_id not in format_filter:
+            continue
+        if guid_filter != [] and rule.uuid not in guid_filter:
+            continue
+        if file_format.pronom_id:
+            formatName = file_format.pronom_id
+            if file_format.pronom_id[:3] == "arc":
                 formatNamespace = "https://archivematica.org"
             else:
                 formatNamespace = "http://www.nationalarchives.uk.gov"
         else:
-            formatName = format.description
+            formatName = file_format.description
```

Comment:
As of commit https://github.com/artefactual-labs/fpr2par/commit/1bb3067097070c4e0ac5f5b9444bdbca8c4ee27c this should now be "slugify(file_format.description)"